### ISSUE TITLE
glide and godep are not working together

### ISSUE THREAD

**Issue by dz0ny** [Masterminds/glide#49]
Heroku is heavily  pushing godep, so the question is if you would be inclined to support baking(like npm shrinkwrap) godeps file based on glide.yml?

**Comment by technosophos** [comment]
That's a good idea. The `glide import godeps` command can parse a Godep file and generate a Glide.yaml, so we could probably go the other way as well. And the Godep format is pretty simple.

**Comment by mattfarina** [comment]
Glide 0.5.0 now has import features in `glide up` and `glide get` when using the `--import` flag. This is in addition to the `glide import` command.

Given the new `vendor` support in Go proper it might be worth updating the heroku buildpack to support that.

**Comment by mattfarina** [comment]
@dz0ny and @technosophos given the latest functionality, is this now no longer an issue? I think™ we can close this.

**Comment by technosophos** [comment]
I'll be testing glide + godep workflows in the next few weeks. Let me find out for sure and get back to you.

**Comment by technosophos** [comment]
I'm thinking this would be a good plugin.

**Comment by edwardmp** [comment]
Any news on this? would be a great featuer

**Comment by technosophos** [comment]
No, we haven't implemented a plugin ourselves. I don't think it would be too hard to do, but it's a little off the beaten path for us, so we probably won't get to it in the very near future.

I know Heroku wants to add Glide support: https://github.com/heroku/heroku-buildpack-go/issues/106, and there's currently a working for (linked from that ticket)
